packages/backend/src/index.ts

```diff
@@ -49,6 +49,7 @@ import {
   LegacyPluginEnvironment as PluginEnvironment,
 } from '@backstage/backend-plugin-manager';
 import { DefaultEventBroker } from '@backstage/plugin-events-backend';
+import { CommonJSModuleLoader } from './loader/CommonJSModuleLoader';
```

Comment:
Please add a comment before this import explaining that this is a temporary fix.
 I'm afraid that we will forget why it is there :-/

Reply:
Added in commit 3976fb9d43f0bee97deb882fd7362827620aaf9e